build.sbt

```diff
@@ -200,26 +200,23 @@ lazy val beam =
 
 Global / onChangedBuildSource := ReloadOnSourceChanges
 
-lazy val pubsub = project
-  .in(file("modules/pubsub"))
+lazy val commonFs2 = project
+  .in(file("modules/common_fs2"))
```

Comment:
It's slightly unusual to use underscores in module names, hyphen should be used.